core/domain/storage_model_audit_jobs_test.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for Oppia storage model audit jobs."""
+import inspect
+
+from core.domain import prod_validation_jobs_one_off
+from core.platform import models
+from core.tests import test_utils
+
+
+# This list includes statistics models and deprecated StateIdMappingModel.
```

Comment:
Shall I also create an issue and assign it to myself and update the issue number here?

Reply:
Yes please. Thanks @ankita240796 !

Reply:
Done in #7101

Reply:
Filed #7100